pages/markets/[marketid].tsx

```diff
@@ -161,46 +162,153 @@ const Market: NextPage<{
   //required to fix title element warning
   const question = indexedMarket.question;
 
-  return (
-    <>
-      <MarketMeta market={indexedMarket} />
-      <div>
-        <div className="flex mb-ztg-33">
-          <MarketImage
-            image={indexedMarket.img}
-            alt={`Image depicting ${question}`}
-          />
-          <div className="sub-header ml-ztg-20">{indexedMarket?.question}</div>
-        </div>
-        <div
-          className="grid grid-flow-row-dense gap-4 w-full "
-          style={{
-            gridTemplateColumns: "repeat(auto-fill, minmax(150px, 1fr))",
-          }}
-        >
-          <Pill
-            title="Ends"
-            value={new Intl.DateTimeFormat("en-US", {
+  const HeaderStat: FC<{ label: string; border?: boolean }> = ({
+    label,
+    border = true,
+    children,
+  }) => {
+    return (
+      <div className={border ? "sm:border-r sm:border-ztg-blue pr-2" : ""}>
+        <span>{label}: </span>
+        <span className="font-medium">{children}</span>
+      </div>
+    );
+  };
+
+  const Tag: FC<{ className?: string }> = ({ className, children }) => {
+    return (
+      <span className={`px-2.5 py-1 rounded bg-gray-300 ${className}`}>
+        {children}
+      </span>
+    );
+  };
+
+  const MarketHeader: FC<{
+    question: string;
+    status: string;
+    tags: string[];
+    createdAt: number;
+    ends: number;
+    prizePool: number;
+    subsidy: number;
+    volume: number;
+    token: string;
+  }> = ({
+    question,
+    status,
+    tags,
+    createdAt,
+    ends,
+    prizePool,
+    subsidy,
+    volume,
+    token,
+  }) => {
+    return (
+      <header className="text-center">
+        <h1 className="font-bold text-4xl my-5">{question}</h1>
+        <div className="flex flex-col sm:flex-row sm:flex-wrap justify-center gap-2 mb-5">
+          <HeaderStat label="Created">
+            {new Intl.DateTimeFormat("en-US", {
               dateStyle: "medium",
-            }).format(Number(indexedMarket.period.end))}
-          />
-          <Pill title="Status" value={indexedMarket.status} />
-          {prizePool ? (
-            <Pill
-              title="Prize Pool"
-              value={`${prizePool} ${store?.config.tokenSymbol}`}
-            />
+            }).format(createdAt)}
+          </HeaderStat>
+          <HeaderStat label={hasDatePassed(ends) ? "Ended" : "Ends"}>
+            {new Intl.DateTimeFormat("en-US", {
```

Comment:
```suggestion
            {new Intl.DateTimeFormat("default", {
```
Best to get locale dynamically